contracts/XVSVault/XVSVaultStorage.sol

```diff
@@ -56,8 +56,8 @@ contract XVSVaultStorageV1 is XVSVaultAdminStorage {
     // Infomation about a withdrawal request
     struct WithdrawalRequest {
         uint256 amount;
-        uint256 lockedUntil;
-        bool afterUpgrade;
+        uint128 lockedUntil;
+        uint128 afterUpgrade;
```

Comment:
It seems first `lockedUntil` and then `afterUpgrade` is the way to go because the EVM is using "[lower-order aligned](https://docs.soliditylang.org/en/v0.8.17/internals/layout_in_storage.html)" to distribute the items of a struct.

This is compatible with how the bytes of each item are stored: big endian.

I did a simple test in Remix to confirm it. I analysed the storage after setting values in a struct similar to the proposed one here:
```
struct WithdrawalRequest {
     uint256 amount;
     uint128 lockedUntil;
     uint128 afterUpgrade;
} 
```
This is the code and the initial storage:
<img width="1724" alt="Screenshot 2023-02-28 at 17 28 56" src="https://user-images.githubusercontent.com/366222/221990847-ec42ee99-f7ba-44bb-b8c7-c7e0a7975adb.png">

After executing the transaction (setting values `1`, `2` and `3` to the struct items), this is the storage:
<img width="639" alt="Screenshot 2023-02-28 at 17 36 16" src="https://user-images.githubusercontent.com/366222/221990922-e442ba46-48e4-4e56-be38-c93e6dcd4108.png">